hazenlib/tasks/task_acr_ghosting.py

```diff
@@ -0,0 +1,194 @@
+"""
+ACR Ghosting
+
+Calculates uniformity for slice 7 of the ACR phantom.
+
+This script calculates the percentage signal ghosting in accordance with the ACR Guidance.
+This is done by first defining a large 200cm2 ROI before placing 10cm2 elliptical ROIs outside the phantom along the
+cardinal directions. The results are also visualised.
+
+Created by Yassine Azma
+yassine.azma@rmh.nhs.uk
+
+14/11/2022
+"""
+
+import sys
+import traceback
+import os
+import numpy as np
+import skimage.morphology
+
+from hazenlib.HazenTask import HazenTask
+
+
+class ACRGhosting(HazenTask):
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def run(self) -> dict:
+        results = {}
+        z = []
+        for dcm in self.data:
+            z.append(dcm.ImagePositionPatient[2])
+
+        idx_sort = np.argsort(z)
+
+        for dcm in self.data:
+            if dcm.ImagePositionPatient[2] == z[idx_sort[6]]:
```

Comment:
This line throws an error why I tested this with the ghosting test data on Hazen --->

index 6 is out of bounds for axis 0 with size 2

Reply:
You need to use the test data in the ACR folder. My scripts are designed to be ran on a folder that contains all 11 slices of a sequence. The ghosting test only looks at the seventh slice.

Reply:
@YassineRMH, checked with the ACR data and it works, thanks!